source/_integrations/refoss.markdown

```diff
@@ -0,0 +1,64 @@
+---
+title: Refoss
+description: Integrate Refoss devices
+ha_category:
+  - Switch
+ha_release: 2023.11
+ha_codeowners:
+  - '@ashionky'
+ha_iot_class: Local Polling
+ha_domain: refoss
+featured: true
+ha_config_flow: true
+ha_platforms:
+  - switch
+ha_integration_type: integration
+---
+
+Integrate Refoss devices into Home Assistant.
+
+## Prerequisites
+
+The device needs to be connected to the local network first. There are two ways to connect devices to the local network:
+  - Using the Refoss app (download and install the Refoss app on Google Play) to connect devices to the local network.
+  - Using the Web page to connect devices to the local network.
+    - Computer/mobile phone connected to device factory wifi.
+    - In the browser, access the address 10.10.10.1, then select the local wifi that needs to be added for device configuration.
+
+{% include integrations/config_flow.md %}
+
+### Discover device
+
+  - After integrating Refoss, will start socket broadcast,Home Assistant will automatically discover Refoss devices under the current LAN.
```

Comment:
```suggestion
  - After integrating Refoss, it will start socket broadcast and Home Assistant will automatically discover Refoss devices under the current LAN.
```